example/CMakeLists.txt

```diff
@@ -1,27 +1,13 @@
-﻿include(CheckCXXCompilerFlag)
-
-if((CMAKE_CXX_COMPILER_ID MATCHES "GNU") OR (CMAKE_CXX_COMPILER_ID MATCHES "Clang"))
-    check_cxx_compiler_flag(-std=c++17 HAS_CPP17_FLAG)
-    if(!HAS_CPP17_FLAG)
-        MESSAGE(FATAL_ERROR "The compiler ${CMAKE_CXX_COMPILER} has no C++17 support.")
-    endif()
-
-    set(CMAKE_VERBOSE_MAKEFILE ON)
-    set(OPTIONS -Wall -Wextra -pedantic-errors -Werror -std=c++17)
-elseif(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
-    check_cxx_compiler_flag(/std:c++17 HAS_CPP17_FLAG)
-    if(!HAS_CPP17_FLAG)
-        MESSAGE(FATAL_ERROR "The compiler ${CMAKE_CXX_COMPILER} has no C++17 support.")
-    endif()
-
-    set(OPTIONS /W4 /WX /std:c++17)
-    if(HAS_PERMISSIVE_FLAG)
-        set(OPTIONS ${OPTIONS} /permissive-)
-    endif()
-endif()
-
-add_executable(example
-                    example.cpp
-                    ${CMAKE_SOURCE_DIR}/include/${CMAKE_PROJECT_NAME}.hpp)
-target_compile_options(example PRIVATE ${OPTIONS})
-target_link_libraries(example PRIVATE ${CMAKE_PROJECT_NAME})
+﻿# set(CMAKE_VERBOSE_MAKEFILE ON)
```

Comment:
Remove this if not need? Or uncomment

Reply:
That is only for debugging. I thought it would be useful sometimes in the future but you are right I isn't strictly required.

I will remove it.

Reply:
It's ok to uncommet this.

Reply:
According to this comment we should not set it.
https://stackoverflow.com/questions/2670121/using-cmake-with-gnu-make-how-can-i-see-the-exact-commands/33741242#comment85451355_33741242

Reply:
Ok, let remove